app/services/validator_check_active_service.rb

```diff
@@ -10,42 +10,22 @@ def initialize
 
   def update_validator_activity
     Validator.includes(:vote_accounts, :validator_score_v1).find_each do |validator|
+      # Skip if validator has no history that is older than DELINQUENT_TIME
+      next if too_young?(validator)
+
+      # mark validator as destroyed if it has no recent history
       if should_be_destroyed?(validator)
```

Comment:
I don't really see a point of keeping this method, since it's the same as simply writing `if validator_histories_for_validator(validator).empty?`, and it's only used once.